src/main/java/org/assertj/core/api/AbstractIterableAssert.java

```diff
@@ -2464,6 +2464,15 @@ public SELF anySatisfy(Consumer<? super ELEMENT> requirements) {
     return myself;
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public SELF noneSatisfy(Consumer<? super ELEMENT> requirements) {
```

Comment:
should we rename `requirements` to `restrictions` ?

Reply:
renamed